build/DependencyVersions.props

```diff
@@ -16,7 +16,7 @@
     <MicrosoftNetCompilersNetcorePackageVersion>$(MicrosoftCodeAnalysisCSharpPackageVersion)</MicrosoftNetCompilersNetcorePackageVersion>
     <MicrosoftNETSdkPackageVersion>2.1.300-preview2-62607-04</MicrosoftNETSdkPackageVersion>
     <MicrosoftNETBuildExtensionsPackageVersion>$(MicrosoftNETSdkPackageVersion)</MicrosoftNETBuildExtensionsPackageVersion>
-    <MicrosoftNETSdkRazorPackageVersion>2.1.0-preview2-30119</MicrosoftNETSdkRazorPackageVersion>
+    <MicrosoftNETSdkRazorPackageVersion>2.1.0-preview2-30169</MicrosoftNETSdkRazorPackageVersion>
```

Comment:
Is there any particular reason you're not just updating all of aspnetcore?

Reply:
I'm specifically trying to unblock some Razor Sdk issues. Feel free to send a PR to update the remainder of aspnetcore if you'd like.

Reply:
On principle, it seems to me like it would be better to keep all packages produced by aspnetcore aligned to the same version instead of mixing various builds.

Reply:
Ok. Is there anything outside of updating the version of `MicrosoftAspNetCoreAppPackageVersion` to do make it work?

Reply:
Nope. MicrosoftAspNetCoreAppPackageVersion defines the version of everything else that comes from aspnet. LZMA, installers, runtime zips, templates, etc.